engine/apps/base/models/user_notification_policy.py

```diff
@@ -66,7 +66,7 @@ def validate_channel_choice(value):
         raise ValidationError("%(value)s is not a valid option", params={"value": value})
 
 
-class UserNotificationPolicyQuerySet(models.QuerySet):
+class UserNotificationPolicyQuerySet(OrderedModelQuerySet):
```

Comment:
a few tests were failing because of:
```python
self = <UserNotificationPolicy: None: Test>
qs = <UserNotificationPolicyQuerySet []>

    def get_ordering_queryset(self, qs=None):
        if qs is None:
            if self.order_class_path:
                model = import_string(self.order_class_path)
                qs = model._meta.default_manager.all()
            else:
                qs = self._meta.default_manager.all()
>       return qs.filter_by_order_with_respect_to(self)
E       AttributeError: 'UserNotificationPolicyQuerySet' object has no attribute 'filter_by_order_with_respect_to'

qs         = <UserNotificationPolicyQuerySet []>
self       = <UserNotificationPolicy: None: Test>

/opt/hostedtoolcache/Python/3.11.3/x64/lib/python3.11/site-packages/ordered_model/models.py:181: AttributeError
```

according to the latest `django-ordered-model` [docs](https://github.com/django-ordered-model/django-ordered-model#custom-manager-and-queryset):

> If your ModelManager returns a custom QuerySet (such as ItemQuerySet below) please have it extend OrderedModelQuerySet, or Django Check E004 will be raised.